contracts/Bridge.sol

```diff
@@ -140,36 +139,12 @@ contract Bridge is Pausable, Context, EIP712 {
         @param handlerAddress Address of handler resource will be set for.
         @param resourceID ResourceID to be used when making deposits.
         @param tokenAddress Address of contract to be called when a deposit is made and a deposited is executed.
+        @param args Additional data to be passed to specified handler.
      */
-    function adminSetResource(address handlerAddress, bytes32 resourceID, address tokenAddress) external onlyAllowed {
+    function adminSetResource(address handlerAddress, bytes32 resourceID, address tokenAddress, bytes calldata args) external onlyAllowed {
```

Comment:
I think we should actually move `tokenAddress` to `args` as it is not always the address of some token. It differs based on the handler type. At least I would rename it not to be tokenAddress.

Reply:
I would avoid that since it would require slicing args in every handler before passing it into `_setResource` function.
`resourceID` and `contractAddress` are always passed into `_setResource` so I would keep them as separate params.